cin_validator/__main__.py

```diff
@@ -79,5 +79,32 @@ def test_cmd(rule, ruleset):
     pytest.main(test_files)
 
 
+@cli.command(name="xmltocsv")
+@click.argument("filename")
+def cli_converter(filename: str):
+    """Converts XML to CSV at selected filepath"""
+    if Path(filename).exists():
+        fulltree = ET.parse(filename)
+        root = fulltree.getroot()
+        data_files = XMLtoCSV(root)
+        cin_tables_dict = {
+            "Header": data_files.Header,
+            "ChildIdentifiers": data_files.ChildIdentifiers,
+            "ChildCharacteristics": data_files.ChildCharacteristics,
+            "ChildProtectionPlans": data_files.ChildProtectionPlans,
+            "CINdetails": data_files.CINdetails,
+            "CINplanDates": data_files.CINplanDates,
+            "Reviews": data_files.Reviews,
+            "Section47": data_files.Section47,
+        }
```

Comment:
We will think of a way to do this without having to spell out all the column names one after the other. It could mean extending the DataContainer mapping used in the run-all CLI command such that it has a keys() method that returns the keys of the mapping. 

Let's put a pin on that.